src/containers/Trips/MulticityTrip.js

```diff
@@ -61,10 +61,11 @@ export class MulticityTrip extends Component {
   componentDidUpdate(prevProps) {
     const { history } = this.props;
     const { multicityTripData } = this.props;
-    if (prevProps.multicityTripData !== multicityTripData) {
-      if (multicityTripData.message === 'Trip requested successfully') {
-        return history.push('/requests');
-      }
+    if (
+        (prevProps.multicityTripData !== multicityTripData) && 
+        (multicityTripData.message === 'Trip requested successfully')
+      ) {
```

Comment:
Can we also check this condition by status instead of message?

Reply:
I forgot to change it.